src/compiler/typescript_to_rust/azle_generate/src/vm_value_conversion/try_from_vm_value_impls/basic.rs

```diff
@@ -170,25 +213,34 @@ pub fn generate() -> proc_macro2::TokenStream {
                             }
                         };
 
+                        // TODO: Consider turning these inner errors into custom errors.
```

Comment:
Make an issue for this

Reply:
Done: https://github.com/demergent-labs/azle/issues/1121